.eslintrc.json

```diff
@@ -4,6 +4,7 @@
         "es6": "true"
     },
     "rules": {
-        "no-template-curly-in-string": "warn"
+        "no-template-curly-in-string": "warn",
+		"semi": ["error", "always"]
```

Comment:
We choose a configuration which was not requiring semi-column (though this can be discussed)

Reply:
I based the style of this on the style of the [`browser-compat-data`](https://github.com/mdn/browser-compat-data) project for consistency.

Reply:
I have no hard feelings against it.